other/run-examples.js

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env node
```

Comment:
can you change the folder to be under '/scripts/run-examples.js' we will eventually add more and this name would be more descriptive